src/build/mod.rs

```diff
@@ -48,6 +49,24 @@ fn rustc_minor_version() -> Option<u32> {
     otry!(pieces.next()).parse().ok()
 }
 
+/// Checks and returns local and latest versions of wasm-pack
+pub fn check_wasm_pack_versions() -> Result<(String, String), Error> {
+    match wasm_pack_local_version() {
+        Some(local) => {
+            match Crate::return_wasm_pack_latest_version() {
+                Some(latest) => Ok((local, latest)),
+                None => Ok((local, "".to_string()))
+            }
+        },
+        None => bail!("We can't figure out what your wasm-pack version is, make sure the installation path is correct.")
```

Comment:
@fitzgen @ashleygwilliams Should we really bail here if we can't determine the user's wasm-pack version? I mean, they can't use the tool at all of this feature fails.

Reply:
This happens in a background thread, and failure just means that we won't print the potential updates message. It crucially should not prevent any other functionality.

Reply:
Ah! Didn't think of this. Thanks for clarifying!